system-tests/projects/dev-server-invalid/cypress/component/support.spec.ts

```diff
@@ -0,0 +1,14 @@
+describe('Support files', () => {
```

Comment:
I wonder if we need this? If not, might be good to remove it and set `supportFile: false` in the `cypress.config.js`, just to keep the example project and files to a minimum.